pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -811,8 +811,14 @@ const (
 
 	// SeedsGroup is the identity group for gardenlets when authenticating to the API server.
 	SeedsGroup = "gardener.cloud:system:seeds"
-	// SeedUserNamePrefix is the identity user name prefix for gardenlets when authenticating to the API server.
+	// SeedUserNamePrefix is the identity username prefix for gardenlets when authenticating to the API server.
 	SeedUserNamePrefix = "gardener.cloud:system:seed:"
+	// ShootsGroup is the identity group for gardenlets running in autonomous shoot clusters when authenticating to the
+	// API server.
+	ShootsGroup = "gardener.cloud:system:shoots"
```

Comment:
I find it counterintuitive that this group is only referring to autonomous shoots, but then the seeds group combines all types of seeds (managed and not managed). I know that autonomous shoots would have a gardenlet installed in them, but still the naming is confusing IMO. If I see this group in logs for example, it would be hard for me to understand its scope without consulting code or documentation. If the group had "autonomous" of "gardenlet" naming embedded it would have been more clear.

That being said, I do not have a concrete suggestion except for changing the shoots group name to something more descriptive, although this would contradict the already established naming used for the seeds group. Food for thought.

Reply:
Good point! What do you think of changing it to `gardener.cloud:system:autonomous-shoots`?

Reply:
Thank you for proposing an alternative! Yes, the proposal would be an improvement IMO. Another option would be `gardener.cloud:system:shoots:autonomous`. I am not sure which one I like better. 😅 

There is also the package and webhook conventions, i.e. `ShootAuthorizer` can become `AutonomousShootAuthorizer`. I have some concerns that names can become longer than needed so no strong opinion on that one. I leave it to your judgement